account_tax_balance/wizard/open_tax_balances.py

```diff
@@ -69,9 +72,11 @@ def open_taxes(self):
             "to": self.to_date.strftime(date_format),
         }
         # name of action which is displayed in breacrumb
-        vals["name"] = self.env._("%(name)s: %(target)s from %(from)s to %(to)s", infos)
+        vals["name"] = (
+            self.env._("%(name)s: %(target)s from %(from)s to %(to)s") % infos  # pylint: disable=W8301
+        )
```

Comment:
The solution should be something like that or similar, not to quiet the linter:

```suggestion
        vals["name"] = (
            self.env._("%(name)s: %(target)s from %(from)s to %(to)s", **infos)
        )
```